api/agent/director.proto

```diff
@@ -129,6 +130,19 @@ message RolloutCommand {
   string command_id = 4;
 }
 
+message SecretSyncCommand {
```

Comment:
Would we have a specific Result message? I didn't see one...

Reply:
The commands arrive by the `Retrieve` method which is a stream which is always open. 

We don't have a result because it's asynchronous. The commands could take some time to execute, and arrive un bulk. 

Although, when a command is finished, the status is reported with a `CommandResult` object via the method `ReportCommandResult`.